src/article/converter/jats/ParagraphConverter.js

```diff
@@ -11,6 +11,14 @@ export default class ParagraphConverter {
   }
 
   import(el, node, importer) {
+    // If the paragraph starts with a LF or CR and then only contains spaces, then ignore it.
+    const child = el.firstChild;
```

Comment:
What do you think about using lodash/get here? 
https://lodash.com/docs/#get

Reply:
Something that I might consider in the future, but I won't bring too much to the table in this case. I've not actually looked at nor used lodash before.